### PR TITLE
[Doc]Add deprecation notice to internal collectors for monitoring

### PR DESCRIPTION
**Pull request by karenzone**
Documentation change for #11511

Merge targets: `7.6`, `7.x`, `master`

PREVIEW HERE:  http://logstash_11526.docs-preview.app.elstc.co/guide/en/logstash/master/configuring-logstash.html

### CODE REVIEW

**Comment by karenzone** [comment]
@andsel   FYI:  Here's the corresponding docs change for #11511

File: docs/static/monitoring/monitoring-internal.asciidoc [hunk 1]
```diff
@@ -2,9 +2,11 @@
 [[monitoring-internal-collection]]
 === Collect {ls} monitoring data using internal collectors
 ++++
-<titleabbrev>Internal collection</titleabbrev>
+<titleabbrev>Internal collection (deprecated)</titleabbrev>
 ++++
 
+deprecated[7.6.0]
```
Comment (andsel) [inline docs/static/monitoring/monitoring-internal.asciidoc]: @karenzone the deprecation starts at `7.7`

**Comment by andsel** [comment]
LGTM just a note on the deprecation starting version

File: docs/static/monitoring/monitoring-internal.asciidoc (hunk 1 above)
Comment (karenzone) [reply docs/static/monitoring/monitoring-internal.asciidoc]: @andsel  Here is my understanding based on our follow-up discussion

* The docs should be updated in 7.6  with the deprecation notice in the docs. 
* Then you PR 11511 to log a deprecation line if the user is using an internal collector goes in 7.7
* Therefore, I should change the version number in line 8 to be 7.7.0.  But I should merge this docs notice for master, 7.x, and *7.6*.

Is this correct?

Comment (andsel) [reply docs/static/monitoring/monitoring-internal.asciidoc]: Yes sounds good, I ping here @roaksoax to double check

Comment (roaksoax) [reply docs/static/monitoring/monitoring-internal.asciidoc]: lgtm!

**Pull request closed without merging**